packages/jwt-verifier/README.md

```diff
@@ -73,6 +75,8 @@ Here is a configuration example that shows the default values:
 ```javascript
 // All values are default files
 const oktaJwtVerifier = new OktaJwtVerifier({
+  issuer: ISSUER,
+  clientId: CLIENTID,
```

Comment:
Nit: We generally use `CLIENT_ID` instead of `CLIENTID`

Reply:
Actually, since these are intended to be default values, we should keep the patten we've outlined above by using `https://{yourOktaDomain}` and `{clientId}`.